### DISCUSSION TITLE
smena-raskladki-kde-alt-shift

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#27] [Blog comments]
# smena-raskladki-kde-alt-shift

Вчера убил пол дня на то, чтобы настроить переключение раскладки клавиатуры на Alt+Shift. Видел очень много решений

https://blog.amd-nick.me/smena-raskladki-kde-alt-shift

**Comment by ShchekodinNA** [comment]
Тут только: мое почтение

**Comment by shockpast** [comment]
😍

**Comment by GDubuar** [comment]
+1  Спасибо

**Comment by grigoriy-st** [comment]
Благодарю!
